espnet2/enh/loss/criterions/time_domain.py

```diff
@@ -76,10 +80,76 @@ def forward(
         return -snr
 
 
+class SDRLoss(TimeDomainLoss):
+    """SDR loss
+
+    filter_length: int
+        The length of the distortion filter allowed (default: ``512``)
+    clamp_db: float
+        clamp the output value in  [-clamp_db, clamp_db]
+    zero_mean: bool
+        When set to True, the mean of all signals is subtracted prior.
+
+    """
+
+    def __init__(self, filter_length=512, clamp_db=None, zero_mean=True):
+        super().__init__()
+        self.filter_length = filter_length
+        self.clamp_db = clamp_db
+        self.zero_mean = zero_mean
+
+    @property
+    def name(self) -> str:
+        return "sdr_loss"
+
+    def forward(
+        self,
+        ref: torch.Tensor,
+        est: torch.Tensor,
+    ) -> torch.Tensor:
+        """args:
+
+        ref: Tensor, (..., n_samples)
+            reference signal
+        est: Tensor (..., n_samples)
+            estimated signal
+
+        Returns
+        -------
+        loss: (...,)
+            the SDR loss (negative sdr)
+        """
```

Comment:
It is better to follow Google's docstring convention as mentioned here: https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#41-python

Reply:
Done.